src/main/java/org/folio/util/OkapiConnectionParams.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.util;
+
+import java.util.Map;
+
+public final class OkapiConnectionParams {
```

Comment:
I think this class is redundant. okapiheaders is already a map and can  be used only to make further calls  or fetch a httpClient when needed

Reply:
Let's stay with this class. It's always easer to extract a header using the predefined method instead of working with HashMap